test/python/qobj/test_pulse_converter.py

```diff
@@ -157,6 +157,20 @@ def test_set_frequency(self):
 
         self.assertEqual(converter(0, instruction), valid_qobj)
 
+    def test_shift_frequency(self):
+        """Test converted qobj from ShiftFrequency."""
+        converter = InstructionToQobjConverter(PulseQobjInstruction, meas_level=2)
+        instruction = ShiftFrequency(8.0, DriveChannel(0))
+
+        valid_qobj = PulseQobjInstruction(
+            name='shiftf',
+            ch='d0',
+            t0=0,
+            frequency=8e-9
```

Comment:
Could we make a similar change to #4399 an make this 8 and the actual qiskit instruction 8e9?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4390/commits/efd2ebaa4456ecc73d8d8324cfbedc629112fc68.